### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by Patrick1904**

### CODE REVIEW

File: src/components/staking/components/ValidatorBox.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ export default function ValidatorBox({
     const dispatch = useDispatch()
     const { accountId: validatorId, active } = validator
 
-    const fee = validator.fee && validator.fee.percentage
+    const fee = Math.round(validator.fee && validator.fee.percentage * 10) / 10
```
Comment (vgrichina) [inline src/components/staking/components/ValidatorBox.js:132]: best to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed to make intent clear

**Review by vgrichina — APPROVED** [review]

**Comment by vgrichina** [comment]
@Patrick1904 looks good, there is one minor comment

File: src/utils/staking.js [hunk 2]
```diff
@@ -301,7 +301,7 @@ export class Staking {
                         contract
                     }
                     const fee = validator.fee = await validator.contract.get_reward_fee_fraction()
-                    fee.percentage = fee.numerator / fee.denominator * 100
+                    fee.percentage = +(fee.numerator / fee.denominator * 100).toFixed(2)
```
Comment (vgrichina) [inline src/utils/staking.js:304]: why is `+` needed?

**Comment by vgrichina** [comment]
@Patrick1904 feel free to merge this, it's already approved

File: src/utils/staking.js (hunk 2 above)
Comment (Patrick1904) [reply src/utils/staking.js:304]: @vgrichina the `+` converts it back to `number` since `toFixed` converts the result into `string`

**Pull request merged**